fenix/app/src/main/res/xml/search_preferences_new.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <PreferenceCategory
+        android:title="Search Engines"
+        android:selectable="false"
+        app:iconSpaceReserved="false"
+        android:layout="@layout/preference_category_no_icon_style">
+        <Preference
+            app:iconSpaceReserved="false"
+            android:key="@string/pref_key_default_search_engine"
+            android:title="@string/preferences_default_search_engine"/>
+        <Preference
+            app:iconSpaceReserved="false"
+            android:key="@string/pref_key_manage_search_shortcuts"
+            android:title="@string/preferences_manage_search_shortcuts"
+            android:summary="@string/preferences_manage_search_shortcuts_summary"/>
+    </PreferenceCategory>
+
+    <PreferenceCategory
+        android:title="@string/preferences_search_address_bar"
+        android:selectable="false"
+        app:iconSpaceReserved="false"
+        android:layout="@layout/preference_category_no_icon_style">
+        <SwitchPreference
+            app:iconSpaceReserved="false"
+            android:defaultValue="true"
+            android:key="@string/pref_key_show_search_engine_shortcuts"
+            android:title="@string/preferences_show_search_engines" />
```

Comment:
**Blocker:** I am seeing the visibility of this preference disappearing when the fragment is being loaded on the device. This is happening as a result of https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/settings/search/SearchEngineFragment.kt#53. I would suggest either removing this or setting this to be default invisible if we don't expect to see this for the new search view.

Otherwise, this will cause some movement when the screen is being created and the preference is being hidden.

Reply:
While I don't like it myself, it's not something that was introduced with the patch. It was copied from `search_preferences.xml` and we have this behaviour in nightly and it has been around for 9 months. 
Created a [follow up](https://bugzilla.mozilla.org/show_bug.cgi?id=1837864).